packages/x-license-pro/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mui/x-license-pro",
-  "version": "6.6.0",
+  "version": "6.9.0",
```

Comment:
We've missed releasing it last week due to https://github.com/mui/mui-x/pull/9135 🙈